### DISCUSSION TITLE
Export data from Statamic to Shopify

### DISCUSSION THREAD

**Discussion by hmalaud** [statamic-rad-pack/shopify#69] [Feature Requests]
Would be nice to be able to (almost) get rid of Shopify's admin once the shop is created and manage all products from Statamic which would update Shopify's shop!

**Comment by jackabox** [comment]
I had a look at this, and I don't think it's in the remit for the plugin at the moment. Two way binding on variants causes various issues due having restrictions in Shopify admin, but not in Statamic.
